packages/common-components/src/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -0,0 +1,174 @@
+import { createStyles, ITheme, makeStyles } from "@chainsafe/common-theme"
+import clsx from "clsx"
+import React, { ReactNode, useCallback, useMemo, useState } from "react"
+import { Typography } from ".."
```

Comment:
This removes a circular dependency warrning.
```suggestion
import { Typography } from "../Typography"
```